typegate/tests/runtimes/python_wasi/python_wasi_test.ts

```diff
@@ -241,3 +241,140 @@ Meta.test("Python WASI: infinite loop or similar", async (t) => {
   //     .on(e);
   // });
 }, { sanitizeOps: false });
+
+Meta.test("Python WASI: typegate reloading", async (metaTest) => {
+  const load = async () => {
+    return await metaTest.engine("runtimes/python_wasi/python_wasi.ts");
+  };
+  const engine = await load();
+  await metaTest.should("work before typegate is reloaded", async () => {
+    // def
+    await gql`
+      query {
+        identityDef(
+          input: {
+            a: "hello",
+            b: [1, 2, "three"]
+          }) {
+          a
+          b
+        }
+      }
+    `
+      .expectData({
+        identityDef: {
+          a: "hello",
+          b: [1, 2, "three"],
+        },
+      })
+      .on(engine);
+
+    // lambda
+    await gql`
+      query {
+        identityLambda(
+          input: {
+            a: "hello",
+            b: [1, 2, "three"]
+          }) {
+          a
+          b
+        }
+      }
+    `
+      .expectData({
+        identityLambda: {
+          a: "hello",
+          b: [1, 2, "three"],
+        },
+      })
+      .on(engine);
+
+    // module import
+    await gql`
+      query {
+        identityMod(input: {
+          a: "hello",
+          b: [1, 2, "three"],
+        }) {
+          a
+          b
+        }
+      }
+    `
+      .expectData({
+        identityMod: {
+          a: "hello",
+          b: [1, 2, "three"],
+        },
+      })
+      .on(engine);
+  });
+
+  // reload typegate
+  const reloadedEngine = await load();
+
+  await metaTest.should("work after typegate is reloaded", async () => {
+    // def
+    await gql`
```

Comment:
maybe refactor a bit ? the code is duplicated.
```
const doReload = async () => { /*...*/ };
await doReload(); // one
await doReload(); // two
```

Reply:
Done. But I have left out the engine reloading part so that it's clear what the test is trying to check.